EIPS/eip-7266.md

```diff
@@ -0,0 +1,46 @@
+---
+eip: 7266
+title: Remove blake2f (0x09) Precompile
+description: Remove the blake2f (0x09) precompile by changing the precompile behaviour to be a dataless revert
+author: Pascal Caversaccio (@pcaversaccio)
+discussions-to: https://ethereum-magicians.org/t/discussion-removal-of-ripemd-160-and-blake2f-precompiles/14857
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-07-03
+---
+
+## Abstract
+
+This EIP removes the [`blake2f`](./eip-152.md) (`0x09`) precompile by changing the precompile behaviour to be a dataless (`0x`) revert.
+
+## Motivation
+
+[EIP-152](./eip-152.md) has never capitalised on a real-world use case. This fact is clearly reflected in the number of times the address `0x09` has been invoked (numbers from the date this EIP was created):
+
+- The most recent call took place on 6 October 2022.
+- Since its go-live as part of the Istanbul hard fork on December 7 2019 (block number 9,069,000), `0x09` has been called only 22,131 times.
```

Comment:
```suggestion
- Since its gone live as part of the Istanbul network upgrade on December 7, 2019 (block number 9,069,000), `0x09` has been called only 22,131 times.
```

Reply:
what's wrong with the BE date format? I use consistently BE...

Reply:
what is BE? I don't usually see dates written in this form without separating year from month and day.

Reply:
Fixed https://github.com/ethereum/EIPs/pull/7266/commits/926d599c5845bd9703ec73b2f9de7f029e3474a6 except for the BE date format. I keep this format style to be consistent.

Reply:
ah british english hehe

Reply:
sure, go ahead with this format then :+1: